### DISCUSSION TITLE
Thrust coefficient in turbine input file

### DISCUSSION THREAD

**Discussion by Marcodep23** [NatLabRockies/floris#927] [General]
Dear all,

I am working with FLORIS (v3.5, with emgauss model) and FAST.Farm trying to achieve the similar results for a really simplified load case: 2 turbines in line (IEA 15MW - Bottom Fixed) spaced by 5D, with an input steady and uniform wind field of 10 m/s. 

To build the turbine input file of FLORIS, I extracted the Ct and Cp values from openfast with a step wind input simulation. However I noticed that the Ct values that I obtained are different from those reported in "IEA-15-240-RWT_tabular.xlsx" present in the path "https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/Documentation" (see figure).

<img width="656" alt="CTcomparison" src="https://github.com/NREL/floris/assets/147702381/a5e0bb86-6ce8-4cec-9a43-6d459ce1b5b5">

Additionally, I have observed that the results in FLORIS, when using the Ct values from the "IEA-15-240-RWT_tabular.xlsx" file, closely match those from FAST.Farm in terms of power and averaged rotor wind speed of the downstream turbine. However, when I use the Ct values calculated by myself, the results in FLORIS differ significantly. This leads me to believe I am making a mistake, but I am unsure what it might be. I am extracting the variable "RtAeroCt/RtFldCt" from OpenFAST.

Hoping to get some help,
Regards,

Marco

**Comment by misi9170** [comment]
Hi Marco,

If I'm understanding your issue correctly, the Ct data that you are getting from your OpenFAST simulations (extracting "RtAeroCt/RtFldCt") is not a great match to the data reported at "https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/Documentation" ---is that correct?

I'm not sure that I can help you here, since we simply implement the Ct curves reported by the OpenFAST team at https://github.com/IEAWindTask37 in FLORIS (we made a dedicated effort to improve this coupling for FLORIS v4, see #771, although the IEA 15MW curves did not change much during this process). However, perhaps you could ask on the [OpenFAST discussions](https://github.com/openfast/openfast/discussions) for the process that is taken to produce the curves in the IEAWindTask37 repository?

Feel free to tag me (@misi9170) if you do open a discussion there! 

Thanks, and sorry to not be very helpful,
Misha

**Reply by rafmudaf** [reply]
Heads up the link to OpenFAST discussions is https://github.com/openfast/openfast/discussions

It's incorrect above, probably just a copy/paste error

**Reply by misi9170** [reply]
Thanks @rafmudaf --- now corrected above!
